src/plugins/credential_management/server/saved_objects/credential_saved_object_type.ts

```diff
@@ -13,40 +13,31 @@ import { SavedObjectsType } from 'opensearch-dashboards/server';
 
 export const credentialSavedObjectType: SavedObjectsType = {
   name: 'credential',
-  namespaceType: 'agnostic',
   hidden: false,
+  namespaceType: 'single',
   management: {
     defaultSearchField: 'title',
     importableAndExportable: true,
     getTitle(obj) {
       return obj.attributes.title;
     },
-    getCredentialType(obj) {
-      return obj.attributes.credential_type;
-    },
-    getUserName(obj) {
-      return obj.attributes.user_name;
-    },
     getEditUrl(obj) {
       return `/management/opensearch-dashboards/credentials/${encodeURIComponent(obj.id)}`;
     },
     getInAppUrl(obj) {
       return {
-        path: `/app/management/opensearch-dashboards/credentials/${encodeURIComponent(obj.id)}`,
-        uiCapabilitiesPath: 'management.opensearchDashboards.credentials',
+        path: `/management/opensearch-dashboards/credentials/${encodeURIComponent(obj.id)}`,
+        uiCapabilitiesPath: 'credential.show',
```

Comment:
not seeing this being used anywhere

Reply:
> not seeing this being used anywhere

https://github.com/elastic-analytics/OpenSearch-Dashboards/blob/2.x/src/plugins/dashboard/server/saved_objects/dashboard.ts#L53